code/controllers/subsystem/air.dm

```diff
@@ -391,8 +391,9 @@ SUBSYSTEM_DEF(air)
 		currentrun.len--
 		EG.breakdown_cooldown++
 		EG.dismantle_cooldown++
-		if(EG.breakdown_cooldown >= EXCITED_GROUP_BREAKDOWN_CYCLES)
+		if(EG.breakdown_cooldown >= EXCITED_GROUP_BREAKDOWN_CYCLES && !(EG.turf_reactions & VOLATILE_REACTION))
 			EG.self_breakdown(poke_turfs = TRUE)
+			EG.dismantle_cooldown = EXCITED_GROUP_BREAKDOWN_CYCLES
```

Comment:
```suggestion
			EG.dismantle_cooldown = min(EG.dismantle_cooldown, EXCITED_GROUP_BREAKDOWN_CYCLES) // Can be delayed
```
sorry I'm stupid

Reply:
I think this might be causing problems with antinoblium replication. I'll investigate it further.

Reply:
I think this stops excited groups from being able to dismantle.